tests/bsim/bluetooth/host/adv/resume2/run.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+import os
```

Comment:
This is not nice :(
Now we have a completely different way of running for a single test. 
This needs to be maintained by other people (usually it is me who needs to do clean ups when something fundamentally different needs to be supported in all tests..)